builders/build-go.ps1

```diff
@@ -0,0 +1,71 @@
+using module "./builders/win-go-builder.psm1"
+using module "./builders/nix-go-builder.psm1"
+
+<#
+.SYNOPSIS
+Generate Go artifact.
+
+.DESCRIPTION
+Main script that creates instance of GoBuilder and builds of Go using specified parameters.
+
+.PARAMETER Version
+Required parameter. The version with which Go will be built.
+
+.PARAMETER Architecture
+Optional parameter. The architecture with which Go will be built. Using x64 by default.
+
+.PARAMETER Platform
+Required parameter. The platform for which Go will be built.
+
+#>
+
+param(
+    [Parameter (Mandatory=$true)][Version] $Version,
```

Comment:
nitpick: please use lowercase for type, to be consistent with other parameters - `[version]`

Reply:
Done.